templates/lp12-module/template/.gitignore

```diff
@@ -0,0 +1,26 @@
+__tmp
+node_modules
+web_modules
+bower_components
+resources
+features
+docs
+vendors
+sandbox
+pids
+logs
+results
```

Comment:
Is this list ignored files list pre-defined by LP? `results`, `features`, `resources` and etc looks very suspicious.